### ISSUE TITLE
Don't run CI on changes to scripts for community management

### ISSUE THREAD

**Issue by JosephTLyons** [zed-industries/zed#19433]
### Check for existing issues

- [X] Completed

### Describe the feature

We no longer run CI for workflow config files used to run scripts to manage community:

- https://github.com/zed-industries/zed/pull/19430

Any script associated with these config files should receive similar treatment. We could move the scripts to a`community` sub-directory and ignore them.

We might want to enable the typos check for changes to the config and script files, like this:

- https://github.com/zed-industries/zed/blob/11a82e3347f1455f51aa59e25684f2c0f4d36068/.github/workflows/docs.yml#L33

**Comment by sikehish** [comment]
I'd like to take this up. Can I be assigned this issue?

**Comment by BLANKatGITHUB** [comment]
I don't think that the workflows will work if you put them in community subdirectory because Github only recognises  workflows located directly at .github/workflows.

You want those workflow to run or completely disable them?
